packages/core/admin/admin/src/content-manager/components/RelationInput/components/Option.js

```diff
@@ -20,6 +20,7 @@ const StyledBullet = styled.div`
 
 export const Option = ({ publicationState, mainField, id }) => {
   const { formatMessage } = useIntl();
+  const mainFieldString = (mainField ?? id).toString();
```

Comment:
I think `stringifiedDisplayValue` is clearer imo, it shows its stringified and `mainField` is already used but this could be the `id` instead